### PR TITLE
Update premium page contents [MAILPOET-4164]

### PR DESCRIPTION
**Pull request by costasovo**
[MAILPOET-4164]

[MAILPOET-4164]: https://mailpoet.atlassian.net/browse/MAILPOET-4164?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mailpoet/views/upgrade.html [hunk 1]
```diff
@@ -224,32 +220,43 @@ <h1 class="mailpoet-h0">
...
+      </h3>
+      <ul class="mailpoet-premium-page-bullet-list">
+        <li><%= _x('Detailed engagement statistics', 'This text resides in the Upgrade page: /wp-admin/admin.php?page=mailpoet-upgrade') %></li>
+        <li><%= _x('Subscriber segmentation', 'This text resides in the Upgrade page: /wp-admin/admin.php?page=mailpoet-upgrade') %></li>
+        <li><%= _x('Google Analytics integration', 'This text resides in the Upgrade page: /wp-admin/admin.php?page=mailpoet-upgrade') %></li>
       </ul>
       <a
         target="_blank"
-        href="https://www.mailpoet.com/premium/agency"
+        href="<%= add_referral_id("https://account.mailpoet.com/?s=" ~ subscriber_count ~ "&g=agency&billing=monthly&email=" ~ current_wp_user.user_email ~ '&utm_source=plugin&utm_medium=premium&utm_campaign=purchase') | escape('html_attr') %>"
```
Comment (JanJakes) [inline mailpoet/views/upgrade.html:257]: Does it need URL escaping? I guess count doesn't and email... shouldn't contain any unsafe characters either?

**Review by JanJakes — COMMENTED** [review]
LGTM but a test is failing. I tried [two reruns](https://app.circleci.com/pipelines/github/mailpoet/mailpoet?branch=update-premium-page&filter=all) (from failed only) and it didn't help.

File: mailpoet/views/upgrade.html (hunk 1 above)
Comment (JanJakes) [reply mailpoet/views/upgrade.html:257]: @costasovo I meant that I don't see URL-escaping. There is `escape('html_attr')` but that's something else. The variables are escaped for HTML attribute but not for URL. But I guess in this case it's safe.

Comment (costasovo) [reply mailpoet/views/upgrade.html:257]: Ahh sry. Now I understand. All those inputs should be safe I guess.

**Review by JanJakes — APPROVED** [review]

**Review by veljkho — APPROVED** [review]

**Pull request merged**